src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
please check if projects outside this repo depending on multiple shared source packages with one or more common dependencies on another shared source package (e.g. an MVC project might use `Microsoft.Framework.CopyOnWriteDictionary.Internal` and `Microsoft.Framework.PropertyHelper.Internal` after updating both to depend on `Microsoft.Framework.NotNullAttribute.Internal`) fail now.  and, see if adding a direct dependency on `Microsoft.Framework.NotNullAttribute.Internal` (in my example) makes the problem better or worse.

my hope is we can be explicit here now.  later we would only need to clean up then-unnecessary dependencies and nothing in these sources once aspnet/XRE#1237 is fixed.